src/main/java/fi/aalto/cs/apluscourses/intellij/services/PluginSettings.java

```diff
@@ -164,16 +181,40 @@ public void initializeLocalSettings() {
    * Resets all local settings to their default values.
    */
   public void resetLocalSettings() {
-    unsetLocalSettings();
-    initializeLocalSettings();
+    unsetLocalIdeSettings();
+    initializeLocalIdeSettings();
   }
 
   /**
-   * Unsets all the local settings from {@link LocalSettingsNames}.
+   * Unsets all the local IDE settings from {@link LocalIdeSettingsNames}.
+   */
+  public void unsetLocalIdeSettings() {
+    Arrays.stream(LocalIdeSettingsNames.values())
+        .map(LocalIdeSettingsNames::getName)
+        .forEach(applicationPropertiesManager::unsetValue);
+  }
+
+  /**
+   * Method (getter) to check the property, representing if the project has been "Turned into A+
+   * project".
+   *
+   * @param project a {@link Project} to set the setting to.
+   * @return a boolean value that represent the state of the project.
+   */
+  public boolean isAPlusProjectSetting(@NotNull Project project) {
+    return Boolean.parseBoolean(
+        PropertiesComponent.getInstance(project).getValue(A_PLUS_IS_A_PLUS_PROJECT.getName()));
+  }
```

Comment:
I accidentally commented on outdated code earlier, but I quite liked the `CourseFileManager.courseFileExists` method, is there a reason to make it a local setting instead?

Reply:
to be fair, I can't remember now... maybe I considered it to be a bit safer than relying on the file existence (can be deleted). but you are right, the brought `CourseFileManager.getInstance().getCourseFile().exists()` could work as well :lobster: 

or maybe [this StackOverflow issue description](https://stackoverflow.com/questions/31350336/within-intellij-ide-the-java-file-exists-always-returns-false) of the problem could have been a reason, I think I had something like that while creating tests

Reply:
I would not change it now...